### ISSUE TITLE
Headers are duplicated for tall tables (mobile/tablet browsers)

### ISSUE THREAD

**Issue by Tako0126** [handsontable/handsontable#4472]
### Description
On a portable device (tablet or phone), column headers are duplicated if the user scrolls down far enough that the original column header disappears.  I believe this is what you call 'floating headers'.  This issue does not appear to be happening on desktop browsers.

### Steps to reproduce
1. Use a tablet or mobile device
2. View a page that has a tall table (simple one is linked below)
3. Scroll down so that the column header(s) disappear
4. Click on a cell
5. A duplicate of the column header(s) will appear at the top of the screen
6. Scroll back up and this temporary column header will remain

### Demo
[Link to repro page](http://htmlpreview.github.io/?https://github.com/Tako0126/Misc/blob/master/Projects/handsontable/defects/defect%20-%20header%20is%20duped%20after%20selecting%20a%20cell.html)

### Your environment
* Handsontable version: I believe 0.34.0.  The repro page just uses the [latest distribution file.](https://handsontable.com/dist/handsontable.full.min.js)
* Browser Name and version: Safari and Chrome, latest versions as of 2017-08-19
* Operating System: iOS

**Comment by AMBudnik** [comment]
Hi @Tako0126 

Handsontable is [not a mobile-friendly library yet](https://docs.handsontable.com/pro/1.13.0/tutorial-known-limitations.html).  You may try if adding `viewportColumnRenderingOffset` and `viewportRowRenderingOffset` helps.

**Comment by Tako0126** [comment]
Thank you for the help.  I tried those settings, but unfortunately they didn't fix the issue.

However, I found a [suggestion](https://www.bountysource.com/issues/3125028-scrolling-problem-floating-header-in-column-sorting-demo) elsewhere to hard-code the following style in the parent container:

`style="height:100%;overflow:auto;"`

This removes the floating column headers, which solves the duplicate header issue for mobile devices. 
 So, looks like there's a good workaround.
